test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -70,7 +76,7 @@ public void PushCommand_PushToV3FileSystemSource()
                 var packageFileName = Util.CreateTestPackage(packageId, version, packageDirectory);
 
                 // Act
-                string[] args = new string[] { "push", packageFileName, "-Source", source };
+                string[] args = new string[] { Command_Push, packageFileName, Param_Source, source };
```

Comment:
As others have commented, I don't think introducing variables helps readability or maintainability, so should be reverted.